staging/src/k8s.io/apimachinery/pkg/util/errors/errors.go

```diff
@@ -147,6 +147,28 @@ func Flatten(agg Aggregate) Aggregate {
 	return NewAggregate(result)
 }
 
+// UniqueAggregateWithCount removes duplicates from flattened Aggregate
+// and returns count for each unique error present in the Aggregate
+func UniqueAggregateWithCount(agg Aggregate) Aggregate {
+	if agg == nil || len(agg.Errors()) == 0 {
+		return agg
+	}
+	errs := Flatten(agg).Errors()
```

Comment:
But as I said that this function UniqueAggregateWithCoun is not specific to scheduler.